[2025-04-25 19:30:17,787][jax._src.callback][ERROR] - jax.io_callback failed
Traceback (most recent call last):
  File "/home/aryaman/anaconda3/envs/cheap/lib/python3.11/site-packages/jax/_src/callback.py", line 436, in io_callback_impl
    return tree_util.tree_map(np.asarray, callback(*args))
                                          ^^^^^^^^^^^^^^^
  File "/home/aryaman/anaconda3/envs/cheap/lib/python3.11/site-packages/jax/_src/callback.py", line 73, in __call__
    return tree_util.tree_leaves(self.callback_func(*args, **kwargs))
                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: make_train.<locals>.train.<locals>.callback() missing 1 required positional argument: 'metric'
